integration/tests/test_sabre.rs

```diff
@@ -76,7 +76,7 @@ fn sabre_cli(command: String) -> Result<Value, TestError> {
     }
 
     if !command_vec.contains(&"--wait") {
-        command_vec.append(&mut vec!["--wait", "300"]);
```

Comment:
This commit probably can be dropped, since the curl-to-wait commit is probably the correct fix